README.md

```diff
@@ -113,9 +117,24 @@ $ pip install --upgrade deepspeech-gpu
 In both cases, it should take care of installing all the required dependencies. Once it is done, you should be able to call the sample binary using `deepspeech` on your command-line.
 
 ```bash
-deepspeech output_model.pb my_audio_file.wav alphabet.txt lm.binary trie
+deepspeech -h
 ```
 
+To actually use the binary without doing your own training, you will need to download the pre-trained models:
+
+```bash
+wget https://github.com/mozilla/DeepSpeech/releases/download/v0.1.0/deepspeech-0.1.0-models.tar.gz
+tar -xvzf deepspeech-0.1.0-models.tar.gz
+```
+
+Then, for an input 16-bit 16 kHz WAV file `my_audio_file.wav`, you can run inferrence:
```

Comment:
This is not completely accurate and already documented somewhere else. We should refer to that instead of copy/pasting the same information.